lib/migration.js

```diff
@@ -161,7 +161,8 @@ function mixinMigration(PostgreSQL) {
         model, self.column(model, propName), actualFields,
       );
       if (!found && self.propertyHasNotBeenDeleted(model, propName)) {
-        sql.push('ADD COLUMN ' + self.addPropertyToActual(model, propName));
+        // add new columns with default values
+        sql.push('ADD COLUMN ' + self.addPropertyToActual(model, propName) + self.columnDbDefault(model, propName));
```

Comment:
If we don't specify default value for a property, should `self.columnDbDefault(model, propName)` be skipped?
Like:

```js
const columnDefaultValue = self.columnDbDefault(model,propName);
if (columnDefaultValue) {
    sql.push('ADD COLUMN ' + self.addPropertyToActual(model, propName) + self.columnDbDefault(model, propName));
} else {
   sql.push('ADD COLUMN ' + self.addPropertyToActual(model, propName))
}
```

Reply:
Yes, if there is no default value set, `self.columnDbDefault(model, propName)` returns `''`.

Reply:
Yeah, it returns an empty string and doesn't impact the final SQL query